python/ray/train/tests/test_lightning_trainer.py

```diff
@@ -112,6 +121,7 @@ def test_trainer_with_ray_data(ray_start_6_cpus_2_gpus, accelerator):
         LightningConfigBuilder()
         .module(cls=LinearModule, input_dim=32, output_dim=4)
         .trainer(max_epochs=num_epochs, accelerator=accelerator)
+        .strategy(strategy)
```

Comment:
Just a question: Any way to assert that model parallel is actually happening? How can we be sure that we set it up correctly?

Reply:
hmm currently we just check if the training process complete without errors.